mern-ecommerce/controllers/order.js

```diff
@@ -61,3 +61,16 @@ exports.updateOrderStatus = (req, res) => {
     }
   );
 };
+
+exports.eraseOrder = async (req, res) => {
+  Order.findByIdAndDelete(req.params.orderId)
+    .then(deletedOrder => {
+      if (!deletedOrder) {
+        return res.status(404).json({ error: "No such order found" });
+      }
+      res.json({ success: "Order has been deleted" });
+    })
+    .catch(error => {
+      res.status(500).json({ error: errorHandler(error) });
```

Comment:
The error response should include a meaningful error message, not just the error itself. It would be better to include a descriptive error message along with the error handler output. For example, `res.status(500).json({ error: 'Internal Server Error', message: errorHandler(error) });`.